lib/dependencies/HarmonyExportImportedSpecifierDependency.js

```diff
@@ -567,10 +567,10 @@ HarmonyExportImportedSpecifierDependency.Template = class HarmonyExportImportedS
 
 				// Filter out exports which are defined by other exports
 				// and filter out default export because it cannot be reexported with *
-				if (activeExports.length > 0)
+				if (activeExports.size > 0)
 					content +=
 						"if(" +
-						JSON.stringify(activeExports.concat("default")) +
+						JSON.stringify(Array.from(activeExports.add("default"))) +
```

Comment:
Technically this is correct, but using the return value of `add` is kind of weird to me.

Better use `JSON.stringify(Array.from(activeExports).concat("default"))`